src/Domain/Entity/Brand.php

```diff
@@ -4,5 +4,5 @@
 
 class Brand extends ProgrammeContainer
 {
-    protected $type = 'brand';
+    const TYPE = 'brand';
```

Comment:
From php7.1 we can specify the visibility so we don't expose these constants publicly `protected const TYPE = 'brand';`